extensions/cornerstone-3d/src/services/SyncGroupService/SyncGroupService.ts

```diff
@@ -96,6 +96,14 @@ export default class SyncGroupService {
         viewportId,
         renderingEngineId,
       });
+
+      // check if any viewport is left in any of syncs, if not delete that sync
```

Comment:
Can we be a bit more particular and verbose. Like `// check if any viewport is left in any of the sync groups, if not, delete that sync group.`

Reply:
yeah I don't know why I put `sync` haha